### PR TITLE
Sensor+DataPoint models added and DataPoint Submission View

### PR DESCRIPTION
**Pull request by Timothyw0**
This closes all 3 issues that were assigned to me

### CODE REVIEW

File: gis_csdt/models.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 from django.utils.timezone import utc
 from django.conf import settings
 from django.db.models import Q#, Count
+from django.contrib.auth.models import User
```
Comment (SillyInventor) [inline gis_csdt/models.py]: From the auth documentation, make sure you get the user by:
from django.contrib.auth import get_user_model
User = get_user_model()

Doing it directly will ignore modifications to the user model, such as we have, and cause problems.

File: gis_csdt/models.py [hunk 2]
```diff
@@ -277,8 +278,14 @@ def __unicode__(self):
 		return self.field_en + ', id:' + str(self.id) + ', dataset:' + str(self.dataset_id)
 
 class Sensor(models.Model):
-	name = models.CharField(max_length=100)
-	sensor_type = models.CharField(max_length=100)
+	name = models.CharField(max_length=100, default='name')
+	supplier = models.CharField(max_length=100, default='supplier')
+        model = models.CharField(max_length=100, default='model')
```
Comment (SillyInventor) [inline gis_csdt/models.py]: Indentation weirdness? Remember to use spaces as is standard in all python apps.

File: gis_csdt/models.py [hunk 3]
```diff
@@ -344,3 +351,15 @@ class Meta:
         elif self.approved and matches.filter(approved=True).count() == 0:
         	matches[0].approved = True
         	matches[0].save()'''
+
+class DataPoint(models.Model):
+	value = models.FloatField()
```
Comment (SillyInventor) [inline gis_csdt/models.py]: Same indentation complaint.

Comment (SillyInventor) [inline gis_csdt/models.py]: Float fields don't handle rounding errors. Please use decimal fields. I am not sure how many decimals you want to allow. models.DecimalField(max_digits=18, decimal_places=15)

**Comment by Timothyw0** [comment]
Migrations are now working successfully but view.py is now complaining about 'DataPoint' is not defined. I'm not sure if this is supposed to happen?

**Comment by SillyInventor** [comment]
@Timothyw0 No, you've made an error.

**Comment by SillyInventor** [comment]
So I have a lot to do, so I have added some tests so that I don't have to check so much.

I haven't added linting (making sure your code is indented ect. correctly) because there is a lot that needs to change. So instead I have added that as something for you to do.

I have also requested that you bring the testing from 30% to 80%, that way it will be even easier for me to go through your code.

Please remember to build and test your code before submitting a PR.

**Comment by SillyInventor** [comment]
@Timothyw0 again, I feel like you're not testing before you submit it. The view you made doesn't even render, much less work. I am going to close this request, and I want you to test and debug it. And THEN open a PR. Testing your work is not optional.

Specifically, I went to /api-datapoint/ and got the error:
image

**Pull request closed without merging**